src/main/java/com/hedvig/productPricing/campaign/service/ReferralServiceImpl.kt

```diff
@@ -44,6 +44,10 @@ constructor(
     val referralCampaign = member.referralCampaign
       ?: throw NullPointerException("No ReferralCampaign for campaignOwner ${member.ownerId}")
 
+    if (referralCampaign.incentive.currency ==  null) {
+      throw java.lang.NullPointerException("Member does not have a currency on ReferralCampaign: {$memberId}")
```

Comment:
Kotlin npe is also fine :) 
```suggestion
      throw NullPointerException("Member does not have a currency on ReferralCampaign: {$memberId}")
```